### ISSUE TITLE
Model weights for Transmitter and Receiver

### ISSUE THREAD

**Issue by parthpatwa** [SivilTaram/Persona-Dialogue-Generation#10]
Can anyone please upload the model weights of trained Transmitter and Receiver? 
Thank you.

**Comment by SivilTaram** [comment]
@parthpatwa Hi, although I do not know why you need weights of Transmitter and Receiver (actually, we have uploaded the final fine-tuned model), I'm uploading it to Google drive. Please wait for minutes.

**Comment by SivilTaram** [comment]
@parthpatwa  Now we only have Receiver for revised mode [LINK](https://drive.google.com/file/d/1c2jSCDXXCoUQWkEOKv9Ga_W0x3pdioJ4/view?usp=sharing) and Transmitter for revised mode [LINK](https://drive.google.com/file/d/109IfzBlAIf90LPTNcMrhWrslkpAi5hpU/view?usp=sharing). Hope it help you.

**Comment by parthpatwa** [comment]
@SivilTaram Thank you. I need it to retrain PSquare on a new data. I will use your existing model weights instead of starting from scratch

These are the steps I am following:
1) initialize transmitter to existing weights and retrain on new data
2)  initialize receiver to existing weights and retrain on new data
3) initialize PSquare to existing weights and retrain by using above obtained transmitter and receiver.

Is this process correct?

**Comment by SivilTaram** [comment]
@parthpatwa Yes, it seems correct for me. And remember to set a small learning rate to avoid catastrophic forgetting (if your own data is relatively small).

**Comment by parthpatwa** [comment]
@SivilTaram help!
the receiver file you uploaded seems to be corrupted. I tried downloading and using 3 times. 
pfa
![image](https://user-images.githubusercontent.com/30171629/92251409-6efc2e00-eeea-11ea-8961-8ad28b339cb0.png)

**Comment by SivilTaram** [comment]
@parthpatwa which version of pytorch do you use? I will recheck if it could be loaded in my environment.

**Comment by parthpatwa** [comment]
@SivilTaram 
I tried on 2 systems-
torch==1.5.1
torchvision==0.6.1

and 
torch==1.6.0
torchvision==0.7.0

both gave same error

**Comment by SivilTaram** [comment]
@parthpatwa You're correct, and the file is corrupted. I have re-uploaded another weight file for receiver [here](https://drive.google.com/file/d/11GWKHpsJNUr1YpVE0lHPNvrA0uKVgpfu/view?usp=sharing). You could try it.
